### PR TITLE
IAS and other cleanup

### PR DESCRIPTION
**Pull request by atharvalele**
- Remove several references to IAS in the docs
- Add container image type to VxWorks doc
- Update Container format image to remove spell check squiggles
- Highlight the default container type
- Deprecate and remove Fastboot, ACRN pages from "How-Tos"

### CODE REVIEW

File: source/how-tos/add-fastboot.rst [hunk 1]
```diff
@@ -3,6 +3,8 @@
 Add Fastboot Support
 ---------------------
 
+.. attention:: Fastboot support is deprecated.
```
Comment (rprangar) [inline source/how-tos/add-fastboot.rst:6]: Please remove this comment

File: source/how-tos/boot-acrn.rst [hunk 2]
```diff
@@ -3,6 +3,8 @@
 Boot ACRN Hypervisor
 ------------------------
 
+.. attention:: ACRN support is deprecated.
```
Comment (rprangar) [inline source/how-tos/boot-acrn.rst:6]: Please remove this comment

File: source/how-tos/add-fastboot.rst (hunk 1 above)
Comment (atharvalele) [reply source/how-tos/add-fastboot.rst]: Removed, thanks!

File: source/how-tos/boot-acrn.rst (hunk 2 above)
Comment (atharvalele) [reply source/how-tos/boot-acrn.rst]: Removed, thanks!

**Review by rprangar — APPROVED** [review]
Looks good

**Pull request merged**